modules/seqexec/web/client/src/main/scala/seqexec/web/client/components/SoundControl.scala

```diff
@@ -16,23 +15,27 @@ import react.semanticui.elements.icon.Icon
 import seqexec.web.client.services.WebpackResources._
 import seqexec.web.client.reusability._
 import web.client.Audio
+import react.common.ReactProps
+
+final case class SoundControl(sound: SoundSelection) extends ReactProps {
+  @inline def render: VdomElement = SoundControl.component(this)
+
+}
 
 /**
   * Button to toggle sound on/off
   */
 object SoundControl {
-  private val SoundOn = Audio.selectPlayable(new Audio(SoundOnMP3.resource),
-                                             new Audio(SoundOnWebM.resource))
-
-  final case class Props(sound: SoundSelection)
+  private val SoundOn =
+    Audio.selectPlayable(new Audio(SoundOnMP3.resource), new Audio(SoundOnWebM.resource))
 
-  implicit val propsReuse: Reusability[Props] = Reusability.derive[Props]
+  implicit val propsReuse: Reusability[SoundControl] = Reusability.derive[SoundControl]
```

Comment:
We can avoid changing all the references to `Props` by using a type alias `type Props = SoundControl`.

Reply:
👍